codecept.multiple.conf.js

```diff
@@ -9,6 +9,23 @@ exports.config = {
   include: {
     I: './test/acceptance/steps_file.js',
   },
+  helpers: {
```

Comment:
Maybe we should merge both configs (base /common and the differences) instead of creating two configs with just a few small changes but many duplicate lines.

Reply:
Once I know this will work, I'll go with optimize it. I just want to make it run with CI, that's my goal now.